lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -892,6 +1017,9 @@ List<html.Element> _clipContent(List<_SaveClipEntry> clipStack,
     curElement = newElement;
     final ui.Rect? rect = entry.rect;
     Matrix4 newClipTransform = entry.currentTransform;
+    final TransformKind transformKind = transformKindOf(
```

Comment:
`transformKindOf` is not cheap, and here it will be applied PAINT_OP_COUNT x CLIP_COUNT times even in pictures with simple transforms, which scares me a bit. Is there a way to apply it once picture-wide?

I suppose we can check the benchmarks.

Reply:
We can optimize if it shows up in benchmarks. canvas.transform is rarer than pushTransform.